### DISCUSSION TITLE
Redash future

### DISCUSSION THREAD

**Discussion by ravikatariyaunzer** [getredash/redash#6874] [General]
"Hello, I've noticed that there hasn't been a version update for Redash in the past two years. Does this mean that Redash is becoming stagnant? I'm considering promoting its use within my organization, but I'm concerned about the lack of a product roadmap for the upcoming years. Can you provide any insight into whether it's advisable to proceed with the current version release?"

**Comment by justinclift** [comment]
Redash re-launched as a "Community Led" project a while back, and we've been working through a bunch off tasks to get the next release official release done.

There is a `preview` image available these days which we point people at, which is pretty good.

I started creating notes for things that need doing with that image after the installation, in order for it to work in real world environments:

https://github.com/getredash/redash/wiki/Using-Redash-%60preview%60

Does that help? :smile:

**Reply by fredymorales** [reply]
@justinclift  Have you considered pushing the preview image to the registry with the tag of the current version as well,  so it's easy to test a pinned version even if it's a dev version ? Maybe you already do it since I see you have it available in the build workflow in https://github.com/getredash/redash/blob/master/.github/workflows/preview-image.yml#L73 but I can't see them listed in the tags https://hub.docker.com/r/redash/redash/tags?page=1&page_size=&ordering=last_updated

and about this:

> working through a bunch off tasks to get the next release official release done.

where can I find those? I'm happy to collaborate if possible 

Thanks!

**Reply by wlach** [reply]
Hey, the preview images are here: https://hub.docker.com/layers/redash/redash/preview/images/sha256-15d5b3d965d3555260238065ada22299822c4eb79feb07314ef041c39c8af3a5?context=explore

**Reply by fredymorales** [reply]
@wlach thanks for the reply but I was wondering if could be possible to include an identifier/version to the preview image so I can pin a specific tag while pulling it, something like: https://hub.docker.com/layers/redash/redash/10.1.0.b50633/images/sha256-f3e8d95656255d9684051604a586dfd50035fa1e297e68379dc1b557f1885c0f?context=explore

**Reply by fredymorales** [reply]
This is what I meant, thank you!
https://github.com/getredash/redash/pull/6965/files#diff-5cd6832cce4678a4fc078932d7ea6e95bee6dca47559d863518bf8730097ea1eR75

**Comment by M-arghandiwal** [comment]
Been using the preview image for a month now, works just fine
